README.md

```diff
@@ -51,7 +58,7 @@ git clone git@github.com:catalyst/moodle-tool_pdfpages.git <moodledir>/admin/too
 
 Use of the converter requires programmatic access, there in no frontend associated with this plugin, so you need to develop another module, or add this plugin to the dependencies of an existing Moodle plugin.
 
-> Only users with the system level capability `tool/pdfpages:generatepdf` can conducted conversions, as this is required to create the single use access key for the headless browser session.
+> Only users with the system or module level capability `tool/pdfpages:generatepdf` can conduct conversions, as this is required to create the single use access key for the headless browser session.
```

Comment:
Updated it to say "in a specific context" xD https://github.com/catalyst/moodle-tool_pdfpages/pull/40/commits/7325fffcf13d151213ca65d758ad063c7125fbb7

The access key is what they are using to login and allow access of target page (I read from the docblock). When we visit the URL of the page we want to print, instead of logging in manually via username/password or SAML, we are using access keys to validate and login the user. Session key is different, that's what we get after we login.